Content.Shared/Storage/EntitySystems/SharedStorageSystem.cs

```diff
@@ -917,7 +917,8 @@ public void UpdateAppearance(Entity<StorageComponent?, AppearanceComponent?> ent
 
         if (TryComp<StorageComponent>(insertEnt, out var insertStorage)
             && GetMaxItemSize((insertEnt, insertStorage)) >= maxSize
-            && !CMStorage.IgnoreItemSize((uid, storageComp), insertEnt))
+            && !CMStorage.IgnoreItemSize((uid, storageComp), insertEnt)
+            && !_whitelistSystem.IsWhitelistPass(storageComp.Whitelist, insertEnt))
```

Comment:
You don't need to change this, use the IgnoreContentsSize component and then set its items whitelist to surgical cases.

Reply:
I see. Thank you for letting me know. Changed.